src/applications/appeals/shared/utils/useBrowserMonitoring.js

```diff
@@ -14,7 +14,7 @@ const defaultRumSettings = {
   // Default settings
   site: 'ddog-gov.com',
   env: environment.vspEnvironment(), // 'vagovprod'
-  sampleRate: 100,
+  sessionSampleRate: 100,
```

Comment:
`sampleRate` was [deprecated in Datadog v4.30.0](https://docs.datadoghq.com/real_user_monitoring/guide/sampling-browser-plans/#setup)